src/App.css

```diff
@@ -73,43 +85,49 @@ span.empty {
     background: #fff; }
 
 .group-0 {
-  box-shadow: inset 0px -1px 9px 1px #ccc; }
+  box-shadow: inset 0px 0px 9px 1px #ccc; }
   .group-0:hover {
     border: 1px solid #ccc; }
 
 .group-1 {
-  box-shadow: inset 0px -1px 9px 1px rgba(226, 60, 60, 0.979); }
+  box-shadow: inset 0px 0px 9px 2px rgba(226, 60, 60, 0.979); }
```

Comment:
To DRY up the group creation look into `mixins` https://sass-lang.com/guide#topic-6

Reply:
This note should be on the scss below